example/fake_api/aiohttp_serpyco.py

```diff
@@ -3,7 +3,7 @@
 import time
 
 from hapic import Hapic, HapicData
-from hapic.error.serpyco import SerpycoDefaultErrorBuilder
+from hapic.error.marshmallow import MarshmallowDefaultErrorBuilder
```

Comment:
why do you add marshmallow in serpyco example?

Reply:
It's a mistake, i replaced by serpyco error builder